tests/adapters/test_epicsadapter/test_epics_adapter.py

```diff
@@ -139,3 +152,31 @@ def test_epics_adapter_load_records_without_DTYP_fields_method(
         written_data = file.read()
 
     assert str(written_data).strip() == str(test_params["expected"]).strip()
+
+
+@pytest.mark.asyncio
+async def test_db_file_not_specified(
+    epics_adapter_no_db_file: EpicsAdapter,
+    MockDevice: Device,
+    mock_raise_interrupt: RaiseInterrupt,
+):
+    epics_adapter_no_db_file.on_db_load = MagicMock()
+    epics_adapter_no_db_file.load_records_without_DTYP_fields = MagicMock()
```

Comment:
For the 4 offending lines id just `# type: ignore`
```suggestion
    epics_adapter_no_db_file.on_db_load = MagicMock()  # type: ignore
    epics_adapter_no_db_file.load_records_without_DTYP_fields = MagicMock() # type: ignore
```